src/main.rs

```diff
@@ -647,116 +632,146 @@ fn main() {
-                                        let e = s.elapsed().as_secs_f32();
-                                        if e > 0.1 {
-                                            info!("socket send took {}s", e);
-                                        }
-                                        if !sent {
-                                            warn!("Send to {} failed", if config.use_wifi {&"tc2-frames server"} else {&"thermal-recorder unix socket"});
-                                            let _ = stream.shutdown().is_ok();
-                                            break 'send_loop;
-                                        } else {
-                                            if let Some(telemetry) = telemetry {
-                                                if let Some(prev_frame_num) = prev_frame_num {
-                                                    if !telemetry.ffc_in_progress {
-                                                        if telemetry.frame_num != prev_frame_num + 1 {
-                                                            // NOTE: Frames can be missed when the raspberry pi blocks the thread with the unix socket in `thermal-recorder`.
-                                                            // println!("====");
-                                                            // println!("Missed {} frames after {}s on", telemetry.frame_num - (prev_frame_num + 1), telemetry.msec_on as f32 / 1000.0);
-                                                        }
-                                                    }
-                                                }
-                                                prev_frame_num = Some(telemetry.frame_num);
-                                                if telemetry.frame_num % 2700 == 0 {
-                                                    info!("Got frame #{}", telemetry.frame_num);
-                                                }
+                                if reconnects > 0 {
+                                    info!("Got frame connection");
+                                    prev_frame_num = None;
+                                    reconnects = 0;
+                                }
+                                let s = Instant::now();
+                                let mut telemetry:Option<Telemetry> = None;
+                                let frame_data = get_frame(is_recording);
+                                if let Some(fb) = frame_data{
+                                    telemetry = Some(read_telemetry(&fb));
+                                    for i in 0..sockets.len(){
+                                        if sockets[i].1.is_some(){
+                                            let stream = sockets[i].1.as_mut().unwrap();
+                                        
+                                            let sent = send_frame( fb,stream);
```

Comment:
My wifi debug feed depended on a different `send_frame` signature and behaviour, which is now broken in this release.  I guess that's fine, I'll fix it next time I need it.